packages/frontend/app/lib/queries.js

```diff
@@ -133,3 +133,15 @@ export const GET_FINANCIAL_STATEMENTS = gql`
     }
   }
 `
+export const GET_TESTIMONIALS = gql`
+  query GetTestimonials {
+    testimonialCollection {
```

Comment:
```suggestion
    testimonialCollection(limit: 8) {
```

This wasn't mentioned in the task, but it'd be great to limit the number of testimonials. Vanessa's already added 17 testimonials 😱 which is way too many for this slider.